proposals/proposal-0001.md

```diff
@@ -0,0 +1,223 @@
+
+The standard proposal workflow is:
+
+- You, as the proposal author, fork the helm/community repository and create a
+  new proposal
+- Proposals are written in [Markdown][markdown]
+- Push your proposal to your GitHub fork and [submit a pull request][pr].
+
+Once the review process is complete, the project maintainers will approve the
+proposal by commenting on the proposal with a "Looks good to me", or "LGTM" for
+short. (note that this is not the same as accepting your proposal!)
+
+The project maintainers will not unreasonably deny a proposal. Reasons for
+denying proposal status include duplication of effort, being technically
+unsound, not providing proper motivation, not addressing backwards
+compatibility, or not in keeping with Helm's design. The core maintainers can be
+consulted during the approval phase.
+
+Once approved, proposal authors are responsible for collecting community
+feedback on a proposal before submitting a feature that satisfies the proposal
+for review. However, wherever possible, long open-ended discussions on public
+mailing lists should be avoided. Strategies to keep the discussions efficient
+include: setting up a separate meetings for the topic, having the proposal
+author accept private comments in the early design phases, setting up a wiki
+page, etc. proposal authors should use their discretion here.
+
+## What belongs in a successful proposal
+
+Each proposal should have the following parts:
+
+1. Preamble - YAML style headers containing metadata about the proposal,
+   including the proposal number, a short descriptive title, the author names,
+   etc.
+1. Abstract - a short (~200 word) description of the technical issue being
+   addressed.
+1. Specification - The technical specification should describe the syntax and
+   semantics of any new feature. If the proposal describes a standard, the
+   specification should be detailed enough to describe how other community
+   members may re-implement the standard in their product.
+1. Motivation - The motivation is critical for proposals that want to change
+   Helm in a significant way. It should clearly explain why existing features
+   are inadequate to address the problem that the proposal solves. proposal
+   submissions without sufficient motivation may be rejected outright.
+1. Rationale - The rationale fleshes out the specification by describing what
+   motivated the design and why particular design decisions were made. It should
+   describe alternate designs that were considered and related work, e.g. how
+   the feature is supported in other projects.
+1. Backwards Compatibility - All proposals that introduce backwards
+   incompatibilities must include a section describing these incompatibilities
+   and their severity. The proposal must explain how the author proposes to deal
+   with these incompatibilities. proposal submissions without a sufficient
+   backwards compatibility treatise may be rejected outright.
```

Comment:
Should SemVer be noted here and how backwards incompatible changes require a major increment of Helm's version.